Makefile

```diff
@@ -1014,7 +1014,7 @@ toplevel/opttoploop.cmx: otherlibs/dynlink/dynlink.cmxa
 make_opcodes := tools/make_opcodes$(EXE)
 
 bytecomp/opcodes.ml: runtime/caml/instruct.h $(make_opcodes)
-	runtime/ocamlrun$(EXE) $(make_opcodes) -opcodes < $< > $@
+	$(CAMLRUN) $(make_opcodes) -opcodes < $< > $@
```

Comment:
`make_opcodes` is generated by `boot/ocamlc -use-prims runtime/primitives`, so it must be run with `runtime/camlrun$(EXE)`.

Reply:
Will address it, thanks and also open a cross compilation PR.

If anyone is interested on why I made that, I got a native cross compiler to work(ocamlopt.opt), and this was a requirement.
https://github.com/EduardoRFS/reason-mobile/blob/master/patches/ocaml/files/make.cross.sh